### ISSUE TITLE
NTR: N-carbamyl-beta-alanine high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8468]
New term request N-carbamyl-beta-alanine high in CSF (elthat_111121052208): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000210
- label: Elevated CSF N-carbamyl-beta-alanine concentration
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: 
- def: The concentration of N-carbamyl-beta-alanine in the cerebrospinal fluid is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8468
- synonyms: N-carbamyl-beta-alanine high in CSF(NONE)
